packages/atomic/src/components/recommendations/atomic-recs-list/e2e/atomic-recs-list.e2e.ts

```diff
@@ -0,0 +1,86 @@
+import {test, expect} from './fixture';
+
```

Comment:
Should we also have tests to validate the behavior when the query returns no recs? I guess we'd just not show the component at all.

Reply:
Sure, that I can do